doc/nrf/drivers/bme68x_iaq.rst

```diff
@@ -0,0 +1,40 @@
+.. _bme68x_iaq:
+
+BME68X IAQ driver
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+You can use the BME68X IAQ driver to run the Bosch Sensor Environmental Cluster (BSEC) library in order to get Indoor Air Quality (IAQ) readings.
+
+.. note::
+   Using the BSEC library requires accepting a separate license agreement.
```

Comment:
Want to be even more precise on that this is a different license and needs to be handled/ downloaded by customer.
Suggested text: 
The BSEC library is distributed with a Bosch proprietary [license](https://www.bosch-sensortec.com/media/boschsensortec/downloads/software/bme688_development_software/2023_04/license_terms_bme688_bme680_bsec.pdf). that blocks it from been distributed as part of our nRF Connect SDK. To start using it you have to accept license and enable the download as described.